### ISSUE TITLE
Chart Panel (Pie Chart) Scrollbar Still Covering Text

### ISSUE THREAD

**Issue by lucashby** [hpcc-systems/Visualization#3580]
This was previously a bug here:  https://github.com/hpcc-systems/Visualization/issues/3488

It was closed, but the issue is not resolved as can be seen in the following example (you may have to manually shrink to see the effect):

```
import { Pie } from "@hpcc-js/chart";
import { ChartPanel } from "@hpcc-js/layout";

const cp = new ChartPanel()
    .widget(new Pie().showSeriesPercentage(true).seriesPercentageFormat(",.2f").showLabels(false))
    .target("target")
    .columns(["Subject", "Year 1"])
    .data([
        ["Other", 151],
["Right Angle", 35],
["Rear End", 34],
["Sideswipe (same direction)", 12],
["Sideswipe (opposite direction)", 9],
["Left Turn (opposite direction)", 8],
["Head On", 7],
["Left Turn (same direction)", 5],
["Right Turn (opposite direction)", 5],
["Right Turn (same direction)", 2]
    ])
    .legendVisible(true)
        .titleVisible(false)
        .legendVisible(true)
        .legend_showSeriesTotal(true)
        .legend_showLegendTotal(true)
        .legend_labelMaxWidth(150)
    .render();
```

**Comment by GordonSmith** [comment]
@lucashby - this fell of the radar, is this still an issue?

**Comment by jbrundage** [comment]
This was fixed in #3711
